ScreenControl_Unity/Assets/ScreenControl/Service/Version.txt

```diff
@@ -0,0 +1,3 @@
+ScreenControl Service Version: {SOFTWARE_VERSION}
+ScreenControl API Version: {API_VERSION}
+Release Reference: {RELEASE_REF}
```

Comment:
Is this actually being used? I cannot see a copy of this with the build artifact

Reply:
Line 73 of the CI is supposed to copy it into the package, so I'll try and fix this 🤔

Reply:
Fixed this in the latest commit, tis in the package now